Update paiement.php

oscim opened the pull request:
mass action payment

# Instructions
*This is a template to help you make good pull requests. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*
*Please:*
- *only keep the "Fix", "Close" or "New" section*
- *follow the project [contributing guidelines](/.github/CONTRIBUTING.md)*
- *replace the bracket enclosed textswith meaningful informations*


# Fix #[*issue_number Short description*]
[*Long description*]


# Close #[*issue_number Short description*]
[*Long description*]


# New [*Short description*]
[*Long description*]

oscim committed:
Update paiement.php

mass action payment

eldy commented on htdocs/compta/paiement.php:
```diff
@@ -63,6 +63,12 @@
     $socid = $user->socid;
 }
+// list of limit id for invoice
+$facids		= GETPOST('facids','alpha');
```
Should use 
GETPOST('facids', 'intcomma');
or a SQL injection will be possible later in the $sql .= ' AND f.rowid IN('.$facids.')';

oscim committed:
Update paiement.php

Fix risque of SQL injection force param list fixed in loop 
$facids		= GETPOST('facids','alpha');
if(empty($facid) && !empty($facids)) {
	$r =explode(',', $facids); 
	$facid = (int)$r[0]; 
}

stickler-ci committed:
Fixing style errors.

eldy commented on htdocs/compta/paiement.php:
```diff
@@ -63,6 +63,12 @@
     $socid = $user->socid;
 }
+// list of limit id for invoice
+$facids		= GETPOST('facids', 'alpha');
```
Should use
GETPOST('facids', 'intcomma');
or a SQL injection will be possible later in the $sql .= ' AND f.rowid IN('.$facids.')';

eldy commented:
It seems this PR was set with "Discussion" or "PR to fix" tag, but no answer nor correction was provided since a long time. So request is automatically closed.
Please reopened and add a comment if you think this is an error.
